suite-native/accounts/src/components/TokenList.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+import { VStack } from '@suite-native/atoms';
+import {
+    EthereumTokenSymbol,
+    selectEthereumAccountsTokensWithBalance,
+} from '@suite-native/ethereum-tokens';
+import { AccountsRootState, selectAccountLabel } from '@suite-common/wallet-core';
+
+import { TokenListItem } from './TokenListItem';
+
+export const TokenList = ({ accountKey }: { accountKey: string }) => {
+    const accountTokens = useSelector((state: AccountsRootState) =>
+        selectEthereumAccountsTokensWithBalance(state, accountKey),
+    );
+    const accountLabel = useSelector((state: AccountsRootState) =>
+        selectAccountLabel(state, accountKey),
+    );
+
+    return (
+        <VStack>
+            {accountTokens?.map((token, index) => (
```

Comment:
I think you should rather do `if(accountTokens) return null;` othewise you will have empty VStack component here, which is not desired.

Reply:
[0a1bb8e](https://github.com/trezor/trezor-suite/pull/7757/commits/0a1bb8e03a8d45650350886eca8a1f4fabdf455a)